bin.js

```diff
@@ -43,8 +44,8 @@ function printVersions(result, print) {
     if (!isSatisfied) {
       var raw = info.wanted.raw;
       var range = info.wanted.range;
-      console.log("Error: Wanted " + name + " version " + raw + " (" + range +")");
-      console.log(PROGRAMS[name].getInstallInstructions(raw));
+      console.error(chalk.red.bold("Error: Wanted " + name + " version " + raw + " (" + range +")"));
+      console.log(chalk.red.bold(PROGRAMS[name].getInstallInstructions(raw)));
```

Comment:
Maybe just the error message should be red and the installation instructions could be normal color? Or are you thinking more red to grab attention?

Reply:
I was thinking more red to grab attention. Otherwise it felt like "info" text along with the rest of the display.

Reply:
Sounds good to me! Thanks for adding this.